backend/src/apiserver/resource/resource_manager.go

```diff
@@ -550,6 +550,13 @@ func (r *ResourceManager) CreateRun(ctx context.Context, run *model.Run) (*model
 	if err != nil {
 		return nil, util.Wrap(err, "Failed to create a run")
 	}
+
+	// Upon run creation, update owning experiment
+	err = r.experimentStore.UpdateLastRun(newRun)
+	if err != nil {
+		return nil, util.Wrap(err, fmt.Sprintf("Failed to update last_run_created_at in experiment %s for run %s", newRun.ExperimentId, newRun.UUID))
+	}
+
```

Comment:
(language nitpicks, feel free to ignore both)

`UpdateLastRun` makes me think we're doing something to the _run_. We're actually just setting a timestamp on the experiment, so I think I would have named it SetLastRunTimestamp or similar.

I'd also say that maybe that log message knows a little too much about what's happening behind closed doors -- maybe leave the column name out ("failed to set last run timestamp on experiment")

Reply:
agree on both, will fix in a follow up pr thanks @gregsheremeta